Editor/Dependencies/Graph/DependencyGraphViewer.cs

```diff
@@ -1,4 +1,4 @@
-#if USE_SEARCH_TABLE
+#if USE_SEARCH_DEPENDENCY_VIEWER
```

Comment:
If you modify this, can you also modify the other places that are related to the dependency graph viewer?

Reply:
Done at https://github.com/Unity-Technologies/com.unity.search.extensions/pull/20/commits/628e2c629302e7b094266f42cfeaa201eeff64a2

Reply:
Extra commit to fix error in ColumnLayoutTests.cs
https://github.com/Unity-Technologies/com.unity.search.extensions/pull/20/commits/c544458088435f5a3469e825200f649eb1b884bf